controllers/operatorconfigmap_controller.go

```diff
@@ -197,8 +204,7 @@ func (c *ClusterVersionReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 	if err := c.own(monConfigMap); err != nil {
```

Comment:
I beleive this is a copy-paste mistake.  
```suggestion
	if err := c.own(encConfigMap); err != nil {
```

Reply:
yes, changed. can resolve?